lua/telescope/utils.lua

```diff
@@ -567,4 +567,45 @@ utils.list_find = function(func, list)
   end
 end
 
+--- Takes the path and parses optional cursor location `$file:$line:$column`
+--- If line or column not present `0` returne returned.
+--- @param path string
+utils.separate_file_path_location = function(path)
```

Comment:
can we also prefix this function with `__`, i am happy to have it as a utils function because then its tested but i am not sure i want extension authors to use that function